.github/CODEOWNERS

```diff
@@ -44,7 +44,7 @@
 /packages/api/src/schemas @dooriya @qinezh @a1exwang @kimizhu
 
 /packages/sdk @tecton @blackchoey @SLdragon @dilin-MS @wenytang-ms @yiqing-zhao
-/packages/sdk/src/conversation @kimizhu @swatDong @a1exwang @XiaofuHuang @dooriya
+/packages/sdk/src/conversation @kimizhu @swatDong @a1exwang @XiaofuHuang @dooriya @SLdragon
```

Comment:
I saw GitHub complains the codeowners file has error, please take a look

Reply:
Seems dilin-MS doesn't have write access to this repo any more, I will remove her.